adjacent_motif_feature_generator.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import itertools
+import logging as log
+
+from common import mutate_string, NUCLEOTIDES
+from generic_feature_generator import GenericFeatureGenerator
+
+class AdjacentMotifFeatureGenerator(GenericFeatureGenerator):
+    """
+    Distances calculated from rightmost nucleotide of query motifs
+    """
+    def __init__(
+            self,
+            motif_len=4,
+            distance_to_right_flank_end=0,
+            feats_to_remove=[],
+            left_update_region=1,
+            right_update_region=1
+        ):
+        """
+        @param motif_len: list of motifs to look for at a distance from mutating position
+        @param distance_to_right_flank_end: distance from mutating position to look for nearby motif
```

Comment:
why is this called `distance_to_right_flank_end`? i thoughts flanks were at the ends of the sequence